.github/workflows/lock-closed.yml

```diff
@@ -1,9 +1,9 @@
-name: Lock issues that are closed and inactive
+name: Lock Inactive Issues
 
 on:
   schedule:
-    # Run at the 40th minute of every hour
-    - cron: '40 * * * *'
+    # Run at 00:00 every day
+    - cron: '0 0 * * *'
```

Comment:
Should we keep them staggered just in case or should it not matter now?

Reply:
I think it should be okay since we are talking about only a few issues a day now.  But we can stagger them to be safe.